build-tools/readme.md

```diff
@@ -0,0 +1,139 @@
+#scraper.js
+
+This script generates Java code representing a type or method defined in `protocol.json`, which can be found [here](https://code.google.com/p/chromium/codesearch#chromium/src/third_party/WebKit/Source/devtools/protocol.json&q=protocol.json&sq=package:chromium&type=cs). The script generates types recursively, keeping track of any additional types that it needs to generate.
```

Comment:
This is way too much documentation for the generic readme.  Move this into `scraper.js` and also document it's usage in the program itself when invoked with no arguments.

This is about discoverability -- you want your documentation to be as close as possible to the actual tool or system they're using, even if it just links externally.